### ISSUE TITLE
Change CWWKW1001W and CWWKW1002W to info messages instead of warnings.

### ISSUE THREAD

**Issue by WhiteCat22** [OpenLiberty/open-liberty#1426]
The following warning messages are displayed when the scope lifecycle of JAX-RS 2.0 and CDI conflicts. They are warning messages and no actions are required.

CWWKW1001W: The scope {1} of JAXRS-2.0 Resource {0} does not match the CDI scope {2}. Liberty gets resource instance from {3}.
This message is displayed if the JAXRS-2.0 resource scope does not match the CDI scope and the resource instance exists in CDI, so Liberty gets the resource instance from CDI. Instance does not include CDI injection if it is from JAXRS.
CWWKW1002W: The CDI scope of JAXRS-2.0 Provider {0} is {1}. Liberty gets the provider instance from {2}.
This message is displayed because provider instance is Singleton only. Liberty gets provider instance from CDI if the CDI scope of provider is Dependent or ApplicationScoped. Instance does not include CDI injection if it is from JAXRS.

Since no action is required, we should change these messages to info to avoid customer confusion.

**Comment by aguibert** [comment]
I don't believe we can change message keys once they have been introduced, since log monitoring utilities typically key off of message keys to watch certain events.  I believe we can change the `W` suffix to an `I` in this case, perhaps with approval from @garypicher

**Comment by garypicher** [comment]
I spoke about this with @NottyCode , and although we could change it in theory, let's get @andymc12 to comment on it first.  It may be that this really is something we want to warn about, and maybe the underlying issue is that some of the text could be clearer.

**Comment by andymc12** [comment]
Sorry for not responding earlier.  My view is that it is worth notifying a customer about because there is  some confusion about the lifecycle of the resource or provider object - i.e. is it per-request or a singleton.  This message serves to inform customers that the lifecycle has been declared in two different ways.  

The JAX-RS spec says that resources should be per-request objects and that providers should be singletons.  However, if the resources or providers are CDI or EJB beans, then it is possible to have a different lifecycle.  So if the customer applied a `@javax.ejb.Singleton` or `@ApplicationScoped` annotation to a resource class with no other changes, then they would see the CWWKW1001W message.  To avoid that they could declare the singleton resources in their `Application` subclass's `getSingleton` method - that tells the JAX-RS runtime that we expect this resource to be treated as a singleton, and then it is in sync with the CDI and/or EJB runtimes.  Likewise, if the customer declared that a provider class was a per-request via CDI or EJB (i.e. `@Stateless` or `@RequestScoped`), then they would need to declare that provider class in the `getClasses` method in order to avoid the CWWKW1002W message.

**Comment by andymc12** [comment]
Per discussion in slack, we agreed that for `jaxrs-2.0` we would leave the current behavior as it is - these will continue to be warning messages.  For `jaxrs-2.1` these messages will not be logged at all by default, but will appear in the trace.log if JAX-RS tracing is enabled.  

This is implemented in pull request #3736.
